ui/src/alerting/components/CheckEOHeader.tsx

```diff
@@ -98,6 +98,10 @@ const CheckEOHeader: FC<Props> = ({
   }
 
   const {singleField, singleAggregateFunc} = isDraftQueryAlertable(draftQueries)
+  const oneOrMoreThresholds =
+    check.type === 'threshold'
+      ? check.thresholds && !!check.thresholds.length
```

Comment:
do we prefer `!!` over `Boolean()`?

Reply:
no strong preference but `!!` feels more readable?